util/logtools/fflogs.ts

```diff
@@ -18,48 +25,146 @@ type ffLogsEventEntry = {
   fight: number;
 };
 
-type fflogsEventResponse = {
+export type fflogsFightListEntry = {
```

Comment:
Are these all fields from fflogs itself (with mismatched camel case and snake case?)

Reply:
Unfortunately yes:
```
{
    "id": 1,
    "boss": 2038,
    "start_time": 21087,
    "end_time": 348888,
    "name": "Rhalgr",
    "zoneID": 1054,
    "zoneName": "Aglaia",
    "size": 24,
    "difficulty": 100,
    "kill": true,
    "partial": 2,
    "inProgress": false,
    "standardComposition": true,
    "hasEcho": false,
    "bossPercentage": 1,
    "fightPercentage": 1,
    "lastPhaseAsAbsoluteIndex": 0,
    "lastPhaseForPercentageDisplay": 0,
    "maps": [
        {
            "mapID": 755,
            "mapName": "Heaven of Lightning",
            "mapFile": "m-z5r1-z5r1.01.jpg"
        }
    ]
}
```

This is the structure for each object in the `events` array that's returned. You can see the full fight response at https://www.fflogs.com:443/v1/report/fights/G48Jt7bcfKVPwyBh?api_key=YOUR KEY HERE